### PR TITLE
Core/Scripts: Ashyen & Keleth Gossip Fix

### PR DESCRIPTION
**Pull request by ghost**
Removes empty gossip text that resulted in [Ashyen](http://www.wowhead.com/npc=17900/ashyen) and [Keleth](http://www.wowhead.com/npc=17901/keleth) saying blank text to players after buffing them.

### CODE REVIEW

**Comment by Kittnz** [comment]
There is no gossip for these in the database?

**Comment by Saben65** [comment]
From what is currently on retail, and from trying to find information during TBC, I can only conclude these guys have never said anything when they buffed players.

**Comment by Saben65** [comment]
Unless it is this...

```SQL
SELECT * FROM `broadcast_text` WHERE `id`=14738;
```
```
    ID  Language  MaleText                                               FemaleText  EmoteID0  EmoteID1  EmoteID2  EmoteDelay0  EmoteDelay1  EmoteDelay2  SoundId    Unk1    Unk2  VerifiedBuild  
------  --------  -----------------------------------------------------  ----------  --------  --------  --------  -----------  -----------  -----------  -------  ------  ------  ---------------
 14738         0  Nature favors those willing to learn its secrets, $n.                     0         0         0            0            0            0        0       0       1            18019
```

**Comment by Saben65** [comment]
Closing. The issue is related to the broadcast_text, not the scripting.

**Comment by Killyana** [comment]
Just add this in a sql file:
```
DELETE FROM `creature_text` WHERE `entry` IN (17900,17901);

File: sql/updates/world/2016_01_18_01_world.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+DELETE FROM `creature_text` WHERE `entry` IN (17900,17901) AND `BroadcastTextId`=8329;
```
Comment (Killyana) [inline sql/updates/world/2016_01_18_01_world.sql]: New line

**Comment by tkrokli** [comment]
Your file "2016_01_18_01_world.sql" needs a new name, because that file name already exists on TC.

**Comment by Carbenium** [comment]
ccf3ed40e2ed5c320fe2f468f5ba0f2bff5e27fa

**Pull request closed without merging**